.eslintignore

```diff
@@ -0,0 +1,9 @@
+static/js/typeahead.bundle.js
+static/js/ical.js
+static/js/calendar.js
+static/js/lazy_load.js
+static/js/little_helpers.js
+static/js/paper_vis.js
+static/js/papers.js
+static/js/persistor.js
+static/js/schedule.js
```

Comment:
I ignored all these files since the changes with prettier on them are pretty intensive and I don't want to introduce accidental bugs.